recipes/vulkan-validationlayers/all/patches/fix-cmake-1.2.198.0.patch

```diff
@@ -0,0 +1,14 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -237,10 +237,7 @@ option(BUILD_LAYERS "Build layers" ON)
+ option(BUILD_LAYER_SUPPORT_FILES "Generate layer files" OFF) # For generating files when not building layers
+ option(USE_ROBIN_HOOD_HASHING "Use robin-hood-hashing" ON)
+ if (USE_ROBIN_HOOD_HASHING)
+-    if(NOT ROBIN_HOOD_HASHING_INSTALL_DIR)
```

Comment:
can we just set `ROBIN_HOOD_HASHING_INSTALL_DIR` (cmake or env var) instead of patching?

Reply:
How would you do that since they hardcode relative path to include dir with `set(ROBIN_HOOD_HASHING_INCLUDE_DIR "${ROBIN_HOOD_HASHING_INSTALL_DIR}/src/include" PATH "Path to robin-hood-hashing/src/include")`?

Reply:
If `ROBIN_HOOD_HASHING_INCLUDE_DIR` was a CACHE variable, it would be possible.

Reply:
I've opened an issue https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/3558

Reply:
the only way I can see is to repackage the `robin hood` layout, so include files are within `src/include`, but it doesn't worth doing that just because of `vulkan-validationlayers`.
they are obviously doing something weird. I didn't notice `src` part hard-coded on the first review.

Reply:
> the only way I can see is to repackage the `robin hood` layout, so include files are within `src/include`, but it doesn't worth doing that just because of `vulkan-validationlayers`.

No, we must not repackage a recipe just because a downstream library is doing bad things.

> they are obviously doing something weird. I didn't notice `src` part hard-coded on the first review.

yes, it's explained in KhronosGroup/Vulkan-ValidationLayers#3558